main.js

```diff
@@ -31,14 +31,23 @@ function setText() {
   wordList = [];
   currentWord = 0;
   correctKeys = 0;
-  inputField.value = '';
   timerActive = false;
   clearTimeout(timer);
   textDisplay.style.display = 'block';
   inputField.className = '';
 
   switch (typingMode) {
+		case 'custom':
+      const helpText = "Paste your custom text and click redo!"
+      wordList = (inputField.value === 'custom' || inputField.value === '') ? helpText.split(" "): betterSplit(inputField.value);
+      punctuation = false;
+			textDisplay.style.height = '3.2rem';

```

Comment:
I think if you change `maxHeight` instead of `height`, you won't have an awkward gap under 'Paste your custom text and click redo!' but instead `textDisplay` will fit content to a MAX of 3.2 rem. This looks cleaner in my opinion.
```suggestion
			textDisplay.style.maxHeight = '3.2rem';

```